test/configure.js

```diff
@@ -0,0 +1,301 @@
+var assert = require( 'chai' ).assert;
+var fs = require( 'fs-extra' );
+var path = require( 'path' );
+
+var sugarcoat = require( '../index' );
+
+suite( 'Configure: Settings', () => {
+
+    test( 'Destination is set to be required. Destination errored out when not supplied.', (done) => {
+
+        var configMissingDest = {
```

Comment:
Since we're testing here a completely empty "settings" object, what do you think about testing for all the errors that are returned?